wp-content/themes/humanity-theme/includes/blocks/header/class-header-block-renderer.php

```diff
@@ -79,6 +86,7 @@ public function __construct( array $attributes = [] ) {
 		);
 
 		$this->image = new Get_Image_Data( $this->attributes['imageID'] );
+		$this->video = new Get_Image_Data( $this->attributes['featuredVideoId'] );
```

Comment:
As the `featuredVideoId` attribute has no default value, this will cause a Fatal Error if no video is set, because the class being instantiated here expects an integer.

Reply:
resolved